src/main/kotlin/org/wycliffeassociates/otter/jvm/app/widgets/ProjectCard.kt

```diff
@@ -1,23 +1,31 @@
 package org.wycliffeassociates.otter.jvm.app.widgets
 
 import javafx.geometry.Pos
+import javafx.scene.control.Button
+import javafx.scene.layout.Pane
 import javafx.scene.layout.VBox
 import org.wycliffeassociates.otter.common.data.model.Project
 import tornadofx.*
 import tornadofx.Stylesheet.Companion.root
 
 class ProjectCard(project: Project) : VBox() {
+    val loadButton = Button()
 
-    val loadProjectButton = button("LOAD")
     init {
         with(root) {
-
             vbox(20) {
                 alignment = Pos.CENTER
-                label(project.book.toString())
-                label(project.targetLanguage.toString())
-                add(loadProjectButton)
+                label(project.book.titleKey)
```

Comment:
don't you need to do a resources lookup?

Reply:
Correct. This was a placeholder since it will change once the new schema/data model classes are implemented. Should we go ahead and add the resources lookup now and just switch out where the key is coming from later?